sos/report/plugins/gluster.py

```diff
@@ -38,9 +38,9 @@ def wait_for_statedump(self, name_dir):
             _spath = self.path_join(name_dir, statedump_file)
             ret = -1
             while ret == -1:
-                with open(_spath, 'r') as sfile:
+                with open(_spath, 'r', encoding='UTF-8') as sfile:
                     last_line = sfile.readlines()[-1]
-                    ret = string.count(last_line, 'DUMP_END_TIME')
+                    ret = last_line.count('DUMP_END_TIME')
```

Comment:
`string` module doesn't have a `count` method. Rather the string class has it.

With the previous code, this would actually be an infinite loop since `ret` will always be -1. I believe this never worked as intended and the plugin typically gets killed on timeout.